system-addon/content-src/components/Search/_Search.scss

```diff
@@ -1,6 +1,6 @@
 .search-wrapper {
   $search-border-radius: 3px;
-  $search-focus-color: var(--newtab-search-focus-border-color);
+  $search-focus-color: $blue-60;
```

Comment:
@Mardak cool with this or any better ideas?

Reply:
Part of it is if a theme wanted to actually change it.. but that's fine for now.